app/assets/javascripts/global_listeners.js

```diff
@@ -99,6 +120,18 @@ $(function() {
   executeAttachinary();
 });
 
+function showNavbar() {
+  $('#navbar').show();
```

Comment:
We could cache all those selectors (like having a `var $navbar = $('#navbar')` and using it instead of querying it from the DOM every time.